code/parachain/frame/assets-registry/src/lib.rs

```diff
@@ -91,8 +92,13 @@ pub mod pallet {
 	/// Mapping local asset to foreign asset.
 	#[pallet::storage]
 	#[pallet::getter(fn from_local_asset)]
-	pub type LocalToForeign<T: Config> =
-		StorageMap<_, Twox128, T::LocalAssetId, ForeignMetadata<T::ForeignAssetId>, OptionQuery>;
+	pub type LocalToForeign<T: Config> = StorageMap<
+		_,
+		Blake2_128Concat,
```

Comment:
why was this changed from `Twox128`?

Reply:
`Twox128` doesn't allow me to `iter()` as `ReversibleStorageHasher` is not satisfied.

Reply:
alocal asset id is permissioed, so i  can be 64 bit hash with concat. i am sure there is such. in other examples.

Reply:
twox128 was overkill

Reply:
you could iterate values and get local if by reverse map. foreign to local and local to foreign

Reply:
we have some assets on rococo, may be on picasso. not big deal now. but recreating them is gated by multisig and democracy.

Reply:
`Twox64Concat` works as well, but still has the problem of changing storage. Should I use that, or go for the `ForeignToLocal-LocalToForeign` double iteration?

Reply:
RPCs have no on-chain cost, so it should be fine to do the double iteration